Bottom: 1d1da945e49647331f04f82a45cc353e9541a6ae
Top:    87aac2094972e0392ac4742a2c54ba76c2f5747a
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-01-18 16:16:02 -0800

Refresh of rsockets-add-support-for-exist

---

diff --git a/src/cma.h b/src/cma.h
index 7135a61..bdd6a85 100644
--- a/src/cma.h
+++ b/src/cma.h
@@ -158,6 +158,7 @@ static inline int ERR(int err)
 }
 
 int ucma_init();
+void ucma_ib_init();
 extern int af_ib_support;
 
 #define RAI_ROUTEONLY		0x01000000
